### ISSUE TITLE
Issue with Inheritance

### ISSUE THREAD

**Issue by JulienM28** [domaindrivendev/Swashbuckle.AspNetCore#1791]
We have a v2 route which use a V2 DTO :

````csharp
 public class TransactionalMailDtoV2 : TransactionalMailDto
    {
        [Required]
        public EmailProvider EmailProvider { get; set; }

        /// <summary>
        /// Paramètres passés dans le Dto
        /// </summary>
        [JsonPropertyName("params")]
        public new Dictionary<string, object> Params { get; set; }
````
But when looking at schemas in Swagger UI it's still the old params Property shown.
![image](https://user-images.githubusercontent.com/23618436/91281241-20cb8a00-e788-11ea-8e25-51af7f5b506a.png)

Thanks

**Comment by domaindrivendev** [comment]
The schema example looks correct to me based on the TransactionalMailDtoV2 class you’ve shown. What are you expecting it to look like?

**Comment by JulienM28** [comment]
````csharp
    public class TransactionalMailDto
    {
             public Dictionary<string, string> Params { get; set; }
````

````csharp
    public class TransactionalMailDtoV2 : TransactionalMailDto
    {
             public new  Dictionary<string, object> Params { get; set; }
````

So I expected 

````json
 "params": {
    "additionalProp1": "string",
    "additionalProp2": "string",
    "additionalProp3": "string"
  }
````
for TransactionalMailDto, it's the case.

And

````json
 "params": {
    "additionalProp1": "object",
    "additionalProp2": "object",
    "additionalProp3": "object"
  }
````
for TransactionalMailDtoV2

**Comment by domaindrivendev** [comment]
working on a fix for this ...

**Comment by domaindrivendev** [comment]
Fixed by 39a150a

**Comment by domaindrivendev** [comment]
Fixed by cbd176b
